source/mainnet/net/guides/create-proofs.rst

```diff
@@ -0,0 +1,148 @@
+
+- Date of birth - Special helper
+- ID valid to
+- ID valid from (maybe?)
+
+Structure a range proof
+-----------------------
+
+In the example below, you see a range proof constructed to check that wallet owner is between 25 and 65 years old (as of today's date).
+
+.. code-block:: console
+
+    {
+      "challenge": "a54bc4116655d247fa625d98f768d4d81e55ffe26ac6bab259bad5895d49ae00",
+      "statement": [
+        {
+          "type": "AttributeInRange",
+          "attributeTag": "dob",
+          "lower": "19571212",
+          "upper": "19971212"
+        }
+      ]
+    }
+
+Membership proofs
+=================
+
+:ref:`Membership proofs<glossary-membership-proof>` allow you to present a list of values to the user's wallet to determine whether the user has one of the required attributes in the list. For example, you might want to know if a user is a resident of the EU. You can present this as a membership proof to be verified by the wallet.
+
+Relevant attributes for membership proofs are:
+
+- Country of residence
+- Country of nationality - Special EU helper
+- Identity document type
+- Identity document issuer
+
+Structure a membership proof
+----------------------------
+
+In the example below, the proof checks that the wallet owner is a citizen of one of the Nordic countries (Finland, Denmark, Sweden, Norway, or Iceland).
+
+.. code-block:: console
+
+    {
+      "challenge": "6c7d69b121d4ce829392d9f2b16395708a458f6183caa20e9074e7283e377418",
+      "statement": [
+        {
+          "type": "AttributeInSet",
+          "attributeTag": "nationality",
+          "set": [
+            "DK",
+            "FI",
+            "IS",
+            "NO",
+            "SE"
+          ]
+        }
+      ]
+    }
+
+Membership proofs can also prove that a user does NOT have an attribute or attributes, in other words a :ref:`non-membership proof<glossary-non-membership-proof>`. For example, if you need to know whether a user is a resident of a country that is subject to trade sanctions and cannot use your service, you might have a proof that determines whether the wallet owner resides in one or more of the countries.
+
+For example, the proof shown below determines if the wallet owner is a citizen of China or North Korea.
+
+.. code-block:: console
+
+    {
+        "challenge": "4de5faf3d68c09e3e76fd8d82ce251c1ff0e49fdcc2661a2f875db35eba02f4d",
+        "statement": [
+          {
+            "type": "AttributeNotInSet",
+            "attributeTag": "nationality",
+            "set": [
+              "CN",
+              "KP"
+            ]
+          }
+        ]
+    }
+
+.. Note::
+
+    Country codes to use for residence and nationality proofs are the `ISO-3166-1 alpha-2 codes <https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2>`_.
+
+Example dApp
+============
+
+Concordium provides the following example demo app and repo: (Still waiting)
+
+The app is a gallery that requires the user to be over 18 years of age to view some content.
```

Comment:
To avoid bad connotations, or misleading implications, would it not be better to make it 13+ or some such